### DISCUSSION TITLE
Milvus hybrid search failed after upgrade from v2.2.11 to v2.3.0

### DISCUSSION THREAD

**Discussion by koveango** [milvus-io/milvus#26983] [Q&A and General discussion]
When I upgrade milvus from v2.2.11 to v2.3.0(CPU), I found that hybrid search api returned plenty of error results like this error message:
```json
{"statusCode":500,"message":"failed to create query plan: cannot parse expression: gender in [ \"MEN\" ] , error: line 1:12 mismatched input '\"MEN\"' expecting {'(', '[', EXISTS, '+', '-', '~', NOT, JSONContains, JSONContainsAll, JSONContainsAny, BooleanConstant, IntegerConstant, FloatingConstant, Identifier, StringLiteral, JSONIdentifier}","error":"Bad Request"}
```
When I restart the standalone node, this error strangely disappeared. But only 1 or 2 hours later, it shows again.

These request could allways be succeed before upgrade process. And when I use other query expr like ` category_id in [ 1,2]`, milvus works well.

So is that there any update of the scalar query format for varchar type fields?

**Comment by xiaocai2333** [comment]
@koveango Can you provide your test code? Seems like a problem that shouldn't arise.

**Reply by koveango** [reply]
@xiaocai2333 I found this error could be easily reproduced on other environment, so I submit an issue.
https://github.com/milvus-io/milvus/issues/26996
